### PR TITLE
tKey API Reference

### PR DESCRIPTION
**Pull request by yashovardhan**
Adds tKey API Reference

### CODE REVIEW

**Comment by yashovardhan** [comment]
Hey @YZhenY @1swaraj @shahbaz17 can I get a final review for this PR before merging? I've made all the changes requested on the last review session.

File: docs/api-reference/self-host/usage.mdx
Comment (shahbaz17) [inline docs/api-reference/self-host/usage.mdx:111]: These lines could be uncommented.

Also, I think these are supposed to go at the bottom of the page.

**Review by shahbaz17 — APPROVED** [review]
LGTM 👍

**Comment by yashovardhan** [comment]
@chaitanyapotti can you review the API reference once?

File: docs/api-reference/self-host/introduction.mdx
Comment (1swaraj) [inline docs/api-reference/self-host/introduction.mdx:18]: Seperate ==> separate

File: docs/api-reference/self-host/introduction.mdx
Comment (1swaraj) [inline docs/api-reference/self-host/introduction.mdx:32]: soley => solely

Comment (yashovardhan) [reply docs/api-reference/self-host/introduction.mdx:32]: fixed

**Comment by yashovardhan** [comment]
@1swaraj done with the changes

**Pull request merged**
